chat-client/src/client/chat.test.ts

```diff
@@ -0,0 +1,69 @@
+import { injectJSDOM } from '../test/jsDomInjector'
+// This needs to be run before all other imports so that mynah ui gets loaded inside of jsdom
+injectJSDOM()
+
+import { afterEach } from 'mocha'
+import { assert } from 'sinon'
+import { NEW_TAB_CREATED, TAB_CHANGED, TAB_REMOVED, UI_IS_READY } from '../contracts/serverContracts'
+import { SEND_TO_PROMPT, TAB_ID_RECEIVED } from '../contracts/uiContracts'
+import { createChat } from './chat'
+import sinon = require('sinon')
+
+describe('Chat', () => {
```

Comment:
Nit: the 'Chat' here gets added to the test descriptions, so you'd get output like:

```
Chat chat initialized publishes ready event
```

Which reads a bit off.